README.md

```diff
@@ -5,13 +5,20 @@ This Repo contains docker-compose scripts to launch EQ
 To use these commands you will need to have Docker installed
 https://docs.docker.com/docker-for-mac/install/
 
-To run all of EQ run:
+### Overview
+
+Here are the various EQ services and how they relate to one another.
+Click each service to navigate to the appropriate GitHub repository.
+
+![EQ Component Diagram](/component-diagram.svg "EQ Component Diagram")
```

Comment:
Is this link correct? when i look at the readme in GitHub it doesn't show the image?